news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+
+In order to avoid frustration and misunderstandings, it was time to come up with a better, more formal and transparent definition for the criteria used to qualify issues’ severity, and how they should be applied. 
+
+The first thing to learn is that severity is to be distinguished from priority. Indeed, severity is used to measure the negative impact that a bug has on a system, a feature, a component or on the project development. It is usually defined by the QA team. As for the priority, it is used to organize all the tasks (bugs, improvements, features, technical tasks) that have to be done in order to meet the project’s deadlines. 
+
+
+## Issue Severity Criteria
+
+When a new issue is created, the first step is to understand what the problem is and then reproduce it. Once that work is done, the second step is to define the severity of that bug.
+
+Four severity levels have been defined, as explained below.
+
+
+### Critical
+
+The bug affects critical functionality or critical data and there is no workaround (no way to avoid it).
+A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
+Examples:
+
+- Difficulty accessing the front office or back office (significant slowdown, error during installation or update, fatal error)
+- Difficulty to globally manage categories, products or customers
+- Difficulty to globally place and manage orders
+
+A critical issue should result in a patch version that should be released as soon as possible. Our recent [spam attack](http://build.prestashop.com/news/fighting-against-spamming-again/) is a good example: it was a critical security issue as our users were flooded with fake customers’ accounts. As a consequence, a patch version has been released that resolve it on 1.6 and 1.7, a good example of a community work.
+
+
+### Major
+
+The bug affects major functionality or major data and there is a workaround (a temporary method for dealing with an issue until a more permanent solution is found), but it is not obvious or can be difficult to put in practice.
+
+A major issue affects a large percentage of users (> 30% and < 60%) and matches at least one of the following:
```

Comment:
```suggestion
A major issue affects a large percentage of users (> 30%) and matches at least one of the following:
```

I'd delete the "< 60%" for clarity's sake. A Major issue for >60% of the people doesn't necessarily become critical IMO.